docs/scenarios_authoring/BasicNodes.md

```diff
@@ -184,3 +183,14 @@ For example, when
 - Output Variable Name is `outputVar`
 
 then nodes that follow `for-each` will be executed twice and the value of current element can be referenced as `#outputVar`. 
+
+## Periodic
+
+![periodic_window](img/periodic.png)
+
+It is mainly testing purposes source providing functionality of sending number of given events every amount of time.
```

Comment:
This source provides functionality of sending a number of given events in a periodic way. It's mainly used for testing.